cdap-ui/app/hydrator/templates/detail.html

```diff
@@ -21,8 +21,9 @@
 <div class="detail-canvas-container" ui-view="canvas"></div>
 
 <!-- TRIGGERS -->
-<div ng-if="$state.params.pipelineId && DetailCtrl.pipelineType === 'cdap-data-pipeline'">
+<div ng-if="$state.params.pipelineId && (DetailCtrl.pipelineType === 'cdap-data-pipeline') || (DetailCtrl.pipelineType === 'cdap-sql-pipeline')">
```

Comment:
minor comment, the parantheses should encapsulate the `||` statements

Reply:
Oops, fixed